src/solvers/solveCobraLPCPLEX.m

```diff
@@ -179,6 +179,7 @@
 %conflict refinement is desired in the case that infeasibility is detected
 %by CPLEX.
 if conflictResolve
+    cpxControl.PARALLEL = 1;

```

Comment:
only to cplex, the function itself if specific to cplex.
this parameter ensures a deterministic solution vector when cplex in multithreaded, In case of conflict, one wants to have the same solution vector to debug, hence the parameter.